File: app/src/main/java/com/example/wellcome/utils/HostingContract.java
package com.example.wellcome.utils;

import android.provider.BaseColumns;

public class HostingContract {
    public HostingContract() { }

    public static class HostingEntry implements BaseColumns {
        public static final String TABLE_NAME = "hosting";
        public static final String COLUMN_NAME_COUNTRY = "country";
        public static final String COLUMN_NAME_CITY = "city";
        public static final String COLUMN_NAME_STREET = "street";
        public static final String COLUMN_NAME_ADDITIONAL_ADDRESS = "additionalAddress";
        public static final String COLUMN_NAME_HOSTER_EMAIL = "hosterEmail";
    }

    public static final String SQL_CREATE_ENTRIES =
            "CREATE TABLE " + HostingEntry.TABLE_NAME + " (" +
                    HostingEntry._ID + " INTEGER PRIMARY KEY," +
                    HostingEntry.COLUMN_NAME_COUNTRY + " TEXT," +
                    HostingEntry.COLUMN_NAME_CITY + " TEXT," +
                    HostingEntry.COLUMN_NAME_STREET + " TEXT," +
                    HostingEntry.COLUMN_NAME_ADDITIONAL_ADDRESS + " TEXT," +
                    HostingEntry.COLUMN_NAME_HOSTER_EMAIL + " TEXT)";

    public static final String SQL_DELETE_ENTRIES =
            "DROP TABLE IF EXISTS " + HostingContract.HostingEntry.TABLE_NAME;
}

File: app/src/main/java/com/example/wellcome/utils/HostingContract.kt
package com.example.wellcome.utils

import com.example.wellcome.utils.HostingContract.HostingEntry
import android.provider.BaseColumns

object HostingContract {
    object HostingEntry : BaseColumns {
        const val TABLE_NAME = "hosting"
        const val COLUMN_NAME_FIRST_NAME = "country"
        const val COLUMN_NAME_LAST_NAME = "city"
        const val COLUMN_NAME_ADDRESS = "street"
        const val COLUMN_NAME_EMAIL = "additionalAddress"
        const val COLUMN_NAME_PHONE = "hosterEmail"
    }

    const val SQL_CREATE_ENTRIES = "CREATE TABLE " + HostingEntry.TABLE_NAME + " (" +
            BaseColumns._ID + " INTEGER PRIMARY KEY," +
            HostingEntry.COLUMN_NAME_FIRST_NAME + " TEXT," +
            HostingEntry.COLUMN_NAME_LAST_NAME + " TEXT," +
            HostingEntry.COLUMN_NAME_ADDRESS + " TEXT," +
            HostingEntry.COLUMN_NAME_EMAIL + " TEXT," +
            HostingEntry.COLUMN_NAME_PHONE + " TEXT)"

    const val SQL_DELETE_ENTRIES = "DROP TABLE IF EXISTS " + HostingEntry.TABLE_NAME
}